src/server/worldserver/worldserver.conf.dist

```diff
@@ -2675,6 +2675,29 @@ Battleground.TrackDeserters.Enable = 1
 
 Battleground.InvitationType = 0
 
+#
+#     Battleground.RewardWinnerHonorFirst
+#     Battleground.RewardWinnerArenaFirst
+#     Battleground.RewardWinnerHonorLast
+#     Battleground.RewardWinnerArenaLast
+#     Battleground.RewardLoserHonorFirst
+#     Battleground.RewardLoserHonorLast
+#        Description: Random Battlegrounds / call to the arms rewards
+#        Default:     30 - Battleground.RewardWinnerHonorFirst
+#                     25 - Battleground.RewardWinnerArenaFirst
+#                     15 - Battleground.RewardWinnerHonorLast
+#                     0  - Battleground.RewardWinnerArenaLast
+#                     5  - Battleground.RewardLoserHonorFirst
+#                     5  - Battleground.RewardLoserHonorLast
+#
+
+Battleground.RewardWinnerHonorFirst = 30
+Battleground.RewardWinnerArenaFirst = 25
+Battleground.RewardWinnerHonorLast  = 15
+Battleground.RewardWinnerArenaLast  = 0
+Battleground.RewardLoserHonorFirst  = 5
+Battleground.RewardLoserHonorLast   = 5
```

Comment:
```suggestion
Battleground.Reward.Winner.Honor.First = 30
Battleground.Reward.Winner.Honor.Last  = 15
Battleground.Reward.Winner.Arena.First = 25
Battleground.Reward.Winner.Arena.Last  = 0
Battleground.Reward.Loser.Honor.First  = 5
Battleground.Reward.Loser.Honor.Last   = 5
```

Putting `.` between shared keywords

Reply:
and there is no reward for "loser.arena.first" ?

Reply:
> and there is no reward for "loser.arena.first" ?

NO

Reply:
ok i commit it if you don't do it

Reply:
It will be added in the future, but not now.